chevah/compat
# [#4681] Add support for UNC and links to Windows Shares.

Scope
=====

This fix tha path handling to work with UNC paths and be able to work with symbolic links to Windows Shares.

Changes
=======

Add support for UNC.
Links to Windows Shares always exist.


How to try and test the changes
===============================

reviewers: @dumol 

This is just to check that compat is green on the supported platform.

Also check that the supported platforms are marked as "Required" in github.

Thanks!
